<?xml version="1.0" encoding="UTF-8"?>
<launch>
<node pkg="masker_util" type="masker_util_node" name="masker_util_mono">
<param name="output_img_topic" value="img_out" />
<remap from="img_in" to="/cam_L/image_raw" />
  </node>
</launch>
